docs/source/fx.experimental.rst

```diff
@@ -0,0 +1,59 @@
+.. currentmodule:: torch.fx.experimental
+
+torch.fx.experimental
+=====================
+
+.. warning::
+   These APIs are experimental and subject to change without notice.
+
+torch.fx.experimental.symbolic_shapes
+-------------------------------------
+.. currentmodule:: torch.fx.experimental.symbolic_shapes
+
+.. autoclass:: torch.fx.experimental.symbolic_shapes.ShapeEnv
```

Comment:
I would recommend using 
```
.. autosummary::
    :toctree: generated
    :nosignatures:

    Class1
    Class2
```
This will generate in this page a one liner about this class based on the first line of the class doc.
And will automatically generate a separate page with all the methods of the class like the classes at the bottom of this section: https://pytorch.org/docs/main/autograd.html#custom-function-utilities